src/BUILD

```diff
@@ -25,8 +25,8 @@ cc_library(
         "include/",
     ],
     deps = [
-        "//external:protobuf",
-    ]
+        "@com_google_protobuf//:protobuf",
```

Comment:
Will this change cause problem with envoy?    e.g. if envoy is using other name for @com_google_protobuf 

Could you try to build envoy with this commit?

Reply:
The names should match since they are based on repo name. I just tried it out anyways, it builds fine.